v0/destinations/marketo/util.js

```diff
@@ -154,6 +166,20 @@ const responseHandler = (destinationResponse, _dest) => {
     "during Marketo Response Handling",
     TRANSFORMER_METRIC.TRANSFORMER_STAGE.RESPONSE_TRANSFORM
   );
+  const { response } = destinationResponse;
+  if (response && !response.success) {
+    // non 2xx failure
+    throw new ErrorBuilder()
+      .setStatus(400)
+      .setMessage(`Error occured during Marketo Response Handling`)
+      .setDestinationResponse(destinationResponse.response)
+      .setStatTags({
+        destType: DESTINATION,
+        stage: TRANSFORMER_METRIC.TRANSFORMER_STAGE.RESPONSE_TRANSFORM,
+        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.EXCEPTION.SCOPE
+      })
+      .build();
+  }
```

Comment:
Should move this to `marketoApplicationErrorHandler`.

Reply:
or rather `marketoResponseHandler`.

Reply:
Moved to `marketoResponseHandler`. Can you please check ?